.github/workflows/ci-workflow.yml

```diff
@@ -64,7 +64,7 @@ jobs:
         run: CC=gcc CXX=g++ make alldebug NUM_THREADS=32
 
       - name: Run test with ASan
-        run: ctest
+        run: ctest --output-on-failure -j 10
```

Comment:
I'm not familiar with ASan. Is it ok to run multiple jobs here?

Reply:
I think it should be fine.

I noticed one test (`CApiConnectionTest.Interrupt`) failing in asan CI test, can you take a further look?

Reply:
I can take a look but this error is also in other PRs.

Reply:
Got it. Can you open an issue on this then?